packages/fxa-graphql-api/src/scripts/must-change-password.ts

```diff
@@ -94,6 +94,8 @@ function adjustText(input = '') {
 
 async function main() {
   const batchSize = program.batchSize ?? 10;
+
+  // TODO: Add Logger and StatsD
```

Comment:
It's not setup here, correct. Did you want to set it up here?

Reply:
Yes, if possible it would be good to add. I guess I wasn't sure about this script though and its usage. I am glad I left this comment and you pointed it out. Are there any caveats here?